MagTag_Weather/magtag_weather.py

```diff
@@ -0,0 +1,272 @@
+import time
+import terminalio
+import displayio
+import adafruit_imageload
+from adafruit_display_text import label
+from adafruit_magtag.magtag import MagTag
+from secrets import secrets
+
+# --| USER CONFIG |--------------------------
+METRIC = False  # set to True for metric units
+# -------------------------------------------
+
+# ----------------------------
+# Define various assets
+# ----------------------------
+BACKGROUND_BMP = "/weather_bg.bmp"
+ICONS_LARGE_FILE = "/weather_icons_70px.bmp"
+ICONS_SMALL_FILE = "/weather_icons_20px.bmp"
+ICON_MAP = ("01", "02", "03", "04", "09", "10", "11", "13", "50")
+DAYS = ("Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday")
+MONTHS = (
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "Augsut",
+    "September",
+    "October",
+    "November",
+    "December",
+)
+magtag = MagTag()
+
+# ----------------------------
+# Backgrounnd bitmap
+# ----------------------------
+bg_bmp, bg_pal = adafruit_imageload.load(BACKGROUND_BMP)
```

Comment:
magtag.set_background doesnt work here?

Reply:
Let me look. Mainly took that approach for more z ordering control of background along with other items.

Reply:
no biggie, i think we sorta overrode the magtag graphics system here, which is ok!

Reply:
Ok, that'll work here. Left over thing from tides, where "background" was more of an intermediate layer thing.